### ISSUE TITLE
Issue with U+5E54(幔)

### ISSUE THREAD

**Issue by KathrynCG** [adobe-fonts/source-han-sans#370]
The shape of CID16799 and CID62253 is nearly the same.
![图片](https://user-images.githubusercontent.com/103583553/211245841-7f5e90e9-7108-447e-8450-c71768f8541d.png)

**Comment by punchcutter** [comment]
No, they are not. Which font and version are you looking at and how? This is the latest version of SourceHanSans-Regular.otf:
<img width="230" alt="uni5e54" src="https://user-images.githubusercontent.com/11257204/211246933-872879fa-ad3c-4456-9868-22ca7b1659ef.png">

**Comment by punchcutter** [comment]
Duh. Sorry, I was looking at the Serif layout file which only has two glyphs and TW uses uni5E54uE0101-JP. I don't see any reason we can't do the same with Sans.

**Comment by punchcutter** [comment]
For 2.005: Removed uni5E54-TW 幔 and remapped to uni5E54uE0101-JP in the HK and TW CMap resources
